app/api/security.py

```diff
@@ -22,8 +22,11 @@ def check_client_id():
         )
 
 
-def verify_token(token: str):
-    """Verify the Google ID token. Raise an HTTPException if the token is invalid."""
+def verify_token(token: str) -> str:
```

Comment:
```suggestion
def verify_and_extract_token(token: str) -> str:
```
or something that makes it clear what is returned